modules/service/src/main/scala/ch/epfl/bluebrain/nexus/iam/service/routes/StaticRoutes.scala

```diff
@@ -18,17 +19,32 @@ import kamon.akka.http.KamonTraceDirectives.traceName
   */
 class StaticRoutes(serviceDescription: ServiceDescription, publicUri: Uri, apiPrefix: String) {
 
-  private val prefix = prefixOf(publicUri)
-
-  def routes: Route = pathPrefix(prefix) {
+  private def serviceDescriptionRoute: Route = pathEndOrSingleSlash {
     get {
-      pathEndOrSingleSlash {
-        traceName("serviceDescription") {
-          complete(Boxed(serviceDescription, List(Link("api", s"$publicUri/$apiPrefix/acls"))))
-        }
+      traceName("serviceDescription") {
+        complete(Boxed(serviceDescription, List(Link("api", s"$publicUri/$apiPrefix/acls"))))
       }
     }
   }
+  private def docsRoute =
+    pathPrefix("docs") {
+      pathEndOrSingleSlash {
+        extractUri { uri =>
+          redirect(uri.copy(path = stripTrailingSlashes(uri.path) / "iam" / "index.html"), StatusCodes.MovedPermanently)
```

Comment:
I think since we have generic nexus docs at `/docs/` we don't need  this redirect.